stats/src/main/java/io/airlift/stats/JmxGcMonitor.java

```diff
@@ -92,7 +112,14 @@ public void stop()
         for (GarbageCollectorMXBean mbean : ManagementFactory.getGarbageCollectorMXBeans()) {
             ObjectName objectName = mbean.getObjectName();
             try {
-                ManagementFactory.getPlatformMBeanServer().removeNotificationListener(objectName, notificationListener);
+                if (mbean.getName().startsWith("ZGC")) {
+                    if (!mbean.getName().equals("ZGC Pauses")) {
```

Comment:
This will miss `ZGC Major Pauses`. We can change to explicit names.

Reply:
My recollection is that this is meant to skip non-generational notifications  (!)

Reply:
Since we are using contains(), I have removed this conditional